src/deluge/gui/views/view.cpp

```diff
@@ -1489,18 +1491,21 @@ void View::pretendModKnobsUntouchedForAWhile() {
 }
 
 void View::cycleThroughReverbPresets() {
-
+	// Get the current value to a tmp variable
 	int32_t currentPreset = getCurrentReverbPreset();
-
-	int32_t newPreset = currentPreset + 1;
+	int32_t newPreset = currentPreset;
+	// Change the tmp value (only if popup is already showing)
+	newPreset = currentPreset + 1;
```

Comment:
I think we can remove references to tmp value here, and just leave it as `int32_t newPreset = currentPreset + 1;`